include/seqan3/alignment/data_structure/aminoacid_scoring_scheme.hpp

```diff
@@ -0,0 +1,355 @@
+        {-2,  1, -7,  2,  8, -6, -4,  0, -6, -6,  1, -6, -4, -1, -2, -2,  3, -1, -1, -2, -2, -4, -6, -5,  6, -2, -8},//E
+        {-4, -6, -4, -6, -6, 10, -6, -2, -1, -1, -5,  0,  0, -6, -3, -6, -5, -5, -4, -4, -3, -2,  0,  4, -6, -3, -8},//F
+        { 0, -2, -6, -3, -4, -6,  9, -4, -7, -7, -3, -7, -5, -1, -3, -5, -4, -4, -1, -3, -3, -6, -6, -6, -4, -3, -8},//G
+        {-3, -1, -7, -2,  0, -2, -4, 12, -6, -6, -1, -5, -4,  1, -2, -4,  1,  0, -2, -3, -2, -5, -4,  3,  0, -2, -8},//H
+        {-3, -6, -2, -7, -6, -1, -7, -6,  7,  5, -5,  2,  2, -6, -2, -5, -5, -5, -4, -2, -2,  4, -5, -3, -6, -2, -8},//I
+        {-3, -7, -3, -7, -6, -1, -7, -6,  5,  5, -5,  4,  3, -6, -2, -5, -5, -5, -4, -3, -2,  3, -5, -3, -6, -2, -8},//J
+        {-1, -1, -6, -2,  1, -5, -3, -1, -5, -5,  8, -4, -3,  0, -2, -2,  2,  3, -1, -1, -2, -4, -6, -4,  1, -2, -8},//K
+        {-3, -7, -3, -7, -6,  0, -7, -5,  2,  4, -4,  6,  3, -6, -2, -5, -4, -4, -4, -3, -2,  1, -4, -2, -5, -2, -8},//L
+        {-2, -5, -3, -6, -4,  0, -5, -4,  2,  3, -3,  3,  9, -4, -2, -4, -1, -3, -3, -1, -2,  1, -3, -3, -3, -2, -8},//M
+        {-3,  5, -5,  2, -1, -6, -1,  1, -6, -6,  0, -6, -4,  9, -2, -4,  0, -1,  1,  0, -2, -5, -7, -4, -1, -2, -8},//N
+        {-1, -3, -4, -3, -2, -3, -3, -2, -2, -2, -2, -2, -2, -2, -2, -3, -2, -2, -1, -1, -2, -2, -5, -3, -1, -2, -8},//O
+        {-1, -4, -6, -3, -2, -6, -5, -4, -5, -5, -2, -5, -4, -4, -3, 12, -3, -3, -2, -3, -3, -4, -7, -6, -2, -3, -8},//P
+        {-2, -1, -5, -1,  3, -5, -4,  1, -5, -5,  2, -4, -1,  0, -2, -3,  9,  1, -1, -1, -2, -4, -4, -3,  5, -2, -8},//Q
+        {-3, -2, -6, -3, -1, -5, -4,  0, -5, -5,  3, -4, -3, -1, -2, -3,  1,  9, -2, -2, -2, -4, -5, -4,  0, -2, -8},//R
+        { 2,  0, -2, -1, -1, -4, -1, -2, -4, -4, -1, -4, -3,  1, -1, -2, -1, -2,  7,  2, -1, -3, -6, -3, -1, -1, -8},//S
+        { 0, -1, -2, -2, -2, -4, -3, -3, -2, -3, -1, -3, -1,  0, -1, -3, -1, -2,  2,  8, -1,  0, -5, -3, -2, -1, -8},//T
+        {-1, -3, -4, -3, -2, -3, -3, -2, -2, -2, -2, -2, -2, -2, -2, -3, -2, -2, -1, -1, -2, -2, -5, -3, -1, -2, -8},//U
+        {-1, -6, -2, -6, -4, -2, -6, -5,  4,  3, -4,  1,  1, -5, -2, -4, -4, -4, -3,  0, -2,  7, -5, -3, -4, -2, -8},//V
+        {-5, -8, -5, -8, -6,  0, -6, -4, -5, -5, -6, -4, -3, -7, -5, -7, -4, -5, -6, -5, -5, -5, 16,  3, -5, -5, -8},//W
+        {-4, -5, -5, -6, -5,  4, -6,  3, -3, -3, -4, -2, -3, -4, -3, -6, -3, -4, -3, -3, -3, -3,  3, 11, -4, -3, -8},//X
+        {-2,  0, -7,  1,  6, -6, -4,  0, -6, -6,  1, -5, -3, -1, -1, -2,  5,  0, -1, -2, -1, -4, -5, -4,  6, -1, -8},//Y
+        {-1, -3, -4, -3, -2, -3, -3, -2, -2, -2, -2, -2, -2, -2, -2, -3, -2, -2, -1, -1, -2, -2, -5, -3, -1, -2, -8},//Z
+        {-8, -8, -8, -8, -8, -8, -8, -8, -8, -8, -8, -8, -8, -8, -8, -8, -8, -8, -8, -8, -8, -8, -8, -8, -8, -8,  1} //*
+        //! [matrix_data blosum80]
+    }};
+
+    //TODO(h-2): add more matrixes
+};
+
+/*!\name Template argument type deduction guides
```

Comment:
In cpp reference it is only called `Type Deduction`. 
I used it as well. I don't care much, but we should have it consistent.

Reply:
Ok, I have grepped our repo and apparently "Type deduction guide(s)" is the most abundant name so I will use that, too, ok?